components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -490,6 +493,10 @@ public RefreshTokenValidationDataDO validateRefreshToken(String consumerKey, Str
                     "TOKEN_SCOPE, TOKEN_STATE, TIME_CREATED FROM " + accessTokenStoreTable +
                     " WHERE CONSUMER_KEY = ? AND REFRESH_TOKEN = ? ORDER BY TIME_CREATED DESC LIMIT 1";
 
+            db2Query = "SELECT ACCESS_TOKEN, AUTHZ_USER, " +

```

Comment:
In the above mentioned PR, I have done a simple fix to it, please refer that. We have moved all the sql queries to SQLQueries.java file since it is the best practise.